### PR TITLE
Extended SR860

### PR DESCRIPTION
**Pull request by ThorvaldLarsen**
Extending SR860 driver to measure conductance and resistance assuming it is used in combination with a current amplifier.

@Dominik-Vogel can you take a look?

### CODE REVIEW

**Comment by jenshnielsen** [comment]
Perhaps it would be better to extend the sr86x driver and then make the sr860_ext a subclass of this extension in the same way as done in the regular drivers. This way we can always do the same for the sr865 and sr865a

**Comment by ThorvaldLarsen** [comment]
Hmm, might be a good idea.

**Comment by ThorvaldLarsen** [comment]
@jenshnielsen does it work this way? I would like to avoid having too many files in the customized instrument folder...

File: qdev_wrappers/customised_instruments/SR86x_ext.py [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+        I = self.X()/self.iv_gain.get_latest()
+        return (V/I)
+
+
+class SR860_ext(SR86x_ext):
+    """
+    The SR860 instrument is almost equal to the SR865, except for the max frequency
+    """
+    def __init__(self, name: str, address: str, reset: bool=False, **kwargs: str) ->None:
+        super().__init__(name, address, max_frequency=5E3, reset=reset, **kwargs)
```
Comment (jenshnielsen) [inline qdev_wrappers/customised_instruments/SR86x_ext.py]: Note this is wrong we just merged https://github.com/QCoDeS/Qcodes/pull/1164 to fix that

Comment (ThorvaldLarsen) [reply qdev_wrappers/customised_instruments/SR86x_ext.py]: Thanks. My local qcodes version seems to be slightly outdated.

**Comment by jenshnielsen** [comment]
Other than that I think it should be fine

**Comment by ThorvaldLarsen** [comment]
Will test that is still works and then merge it.

**Pull request merged**